include/TemplatesMenu.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TEMPLATESMENU_H
+#define TEMPLATESMENU_H
+
+#include <QDir>
+#include <QMenu>
+
+class TemplatesMenu : public QMenu
+{
+	Q_OBJECT
+public:
+	TemplatesMenu(QWidget *parent = nullptr);
+	virtual ~TemplatesMenu() = default;
+
+private slots:
+	void createNewProjectFromTemplate( QAction * _idx );
+	void fillTemplatesMenu();
+	int addTemplatesFromDir( QDir dir );
+
+private:
+	int m_custom_templates_count;
```

Comment:
I see there isn't anything about variable names other than starting with `m_` in the coding conventions wiki page, but this variable doesn't follow the camelCase that I see everywhere else. I realize it was a refactoring, but perhaps this would be a good time to specify variable name styling as well?

Reply:
I have used camelCase here now.

In general this is something clang-tidy could catch (https://github.com/LMMS/lmms/pull/4690).